doc/08-advanced-topics.md

```diff
@@ -484,7 +484,7 @@ host or service is considered flapping until it drops below the low flapping thr
 The attribute `flapping_ignore_states` allows to ignore state changes to specified states during the flapping calculation.
 
 `FlappingStart` and `FlappingEnd` notifications will be sent out accordingly, if configured. See the chapter on
-[notifications](alert-notifications) for details
+[notifications](03-monitoring-basics#notifications) for details
```

Comment:
The link should include the `.md` extension, otherwise it's broken when [viewed on GitHub](https://github.com/Icinga/icinga2/blob/122c89f4cd558379765ed822ceece394416278e1/doc/08-advanced-topics.md#check-flapping).

Reply:
fixed